v2/internal/frontend/desktop/linux/window.go

```diff
@@ -766,6 +772,15 @@ func (w *Window) IsMaximised() bool {
 	return result > 0
 }
 
+func (w *Window) IsMinimised() bool {
+	result := C.IsMinimised(w.asGTKWidget())
```

Comment:
Just wondering if this needs to be run on the main thread...

Reply:
I have not tested this yet. I am currently testing this in an Ubuntu VM.

Reply:
Hey @zandercodes 👋 Just following up on the Linux testing. I can help with this if you need? It would be good to get this in!

Reply:
I've gone through and resolved the conflict. This should be good but let me know if there are issues 👍

Reply:
Hi @leaanthony, at the moment I don't have time to test this on Linux. But I will do it this week.

Reply:
Thanks. I might be able to. Will update here